src/main/java/org/parabot/core/asm/wrappers/Getter.java

```diff
@@ -77,4 +77,21 @@ public AddGetterAdapter getAdapter() {
         return new AddGetterAdapter(into, fieldLocation, fieldNode, methodName, returnDesc, staticMethod, multiplier);
     }
 
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
```

Comment:
String.format would be better here too

Reply:
What would be the easiest way of doing it because it needs a null check on fieldNode?

Reply:
Missed that part, you're right, here it's fine